pkgs/development/python-modules/odoorpc/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "odoorpc";
+  version = "0.10.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "OCA";
+    repo = "odoorpc";
+    tag = "v${version}";
+    hash = "sha256-59txgRhZ6CWCamv3BVSGDDZ5Q1RDW7wC1TEsen/OD6M=";
+  };
+
+  build-system = [ setuptools ];
+
+  pythonImportsCheck = [ "odoorpc" ];
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
nix-update-script isn't necessary in `buildPython*`. Its auto included.

Reply:
thanks; attended